histolab/filters/compositions.py

```diff
@@ -139,3 +142,35 @@ def tissue_mask_filters(self) -> imf.Compose:
                 mof.BinaryFillHoles(structure=np.ones((20, 20))),
             ]
         )
+
+
+class _CustomFiltersComposition(FiltersComposition):
+    def __init__(
+        self, cls_: Type[imf.Compose], *custom_filters: imf.ImageFilter
+    ) -> None:
+        """Create a custom filter composition with at least one filter.
+
+        Generally, their is no need to call this constructor manually.
+        Please use ``FiltersComposition(Compose, *filters)`` instead.
+        """
+
+        if cls_ != imf.Compose:
+            raise FilterCompositionError("cls_ parameter cannot be None")
```

Comment:
as Codecov says, this line is not covered by tests (in this case it should be a unit test)